service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ClassCode.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum ClassCode {
```

Comment:
ClassCode is a rather generic name. Does it refer to CompundStatement only or is it common in all MIM elements?

Reply:
Related to `CompoundStatement` mainly, I can update that to make it clearer